### DISCUSSION TITLE
第一回議事録

### DISCUSSION THREAD

**Discussion by unnnyou** [UNNNYOU/Project-RQB#4] [議事録]
## はじめに 当初案を共有
当初の検討していたものを箇条書きします。

### 当初検討案

- RUNTEQ版メンタを作成
- 特定の技術を使ったことがある方とその技術で悩んでいる人をマッチングさせる

### 作成時のメリット

- 課題に対する解決やRUNTEQ生同士での助け合いの効率化
- 一部の相談の受けている方の負担軽減
- メンターの経験を積める

### 作成想定期間

１ヶ月程度

#### フロント　NEXT.js

- UIコンポーネント：未定

#### バック　Ruby on Rails 

- 通知機能
  - マタモ⇨マタモAPI
  - GitHub⇨OmniAuthかOctikitあたり
  - LINE, Discord⇨OmniAuth

- 認証他
  - SorceryかDeviseか自前実装
- DM系
  - WebSocketあたり

**Comment by unnnyou** [comment]
## メンタのようなアプリについて
かなり懸念点があり、現時点ではメンタのようなアプリは理想的ではなさそう。

#### 懸念点

- 現在、特定のGiverによって奇跡的に成り立っている。そのため、そのような状態を土台にするのは危険である。
- 悪質なTakerによって、Giveをする人がいなくなった場合にはアプリ自体が使用されなくなる。
- メンターのような１体１でのやり取りではメンターの負担がでかい
- メンターにはメリットがないため、善意に頼らざる終えない

#### この内容のまとめ

Giverによる善意の行動に依存してしまうため、アプリが使用されなくなる可能性が高い。
そのGiver自体も特定の人物に限られてしまう可能性があり、その場合にはその人物への負担が大きくなってしまう。

また、悪質なTakerによってGiveが嫌になってしまう可能性もある。
そのため、メンタのようなアプリは金銭などの報酬が発生しない限り難しいのではないか？
と結論づけた。

**Comment by unnnyou** [comment]
## 質問掲示板アプリについて
現時点ではこちらへシフトしようとしている。
カリキュラム外で発生した問題に対して、RUNTEQ生同士で助け合うことができる。
質問が集中しているRUNTEQ生の負担軽減になる可能性がある。

### アプリ概要

RUNTEQ生同士での[stack overflow](https://stackoverflow.co/)のような質問掲示板アプリ。

アプリ作成時に発生している問題についての質問投稿を行い、それに対して回答する形式。
回答者は自分が回答できそうな内容か、また回答したいかどうかを任意で判断することができる。

### メリット

- カリキュラム外での問題をRUNTEQ生同士で対応できる
- コミュニティに入るのが怖い人でも使いやすい可能性がある
- 回答者が特定の人物に集中しないため、特定の人物の負担が大きくなることはない
- 評価制度などを導入することで質問力、回答力を高められる。
- 同じようなエラーに遭遇した時の参考になる。

### デメリット

- Taker自体は出てきてしまう可能性がある
- 質問自体が意味がわからない可能性がある

**Comment by unnnyou** [comment]
## 総括

カリキュラム外の問題に対してRUNTEQ生同士での助け合いという意味では需要のあるアプリだと考えられる。
問題点自体はあるが、適切な運用を行なっていけばRUNTEQ生にとって良いアプリになる可能性が高い。

今回の内容では、メンタのようなアプリではなく質問掲示板のようなアプリを作成し、色々な方を救済していければいいのではないか？という内容でまとまった。

**Comment by unnnyou** [comment]
ネムネムモードで書いたから変なところとか修正点があれば教えてください。

**Reply by topi-log** [reply]
綺麗にまとめられていて見やすく助かります
内容大丈夫だと思います！！
（ちなみに「stack over flow」=>「[stack overflow](https://stackoverflow.co/)」か「Stack Overflow」かもです）

**Reply by unnnyou** [reply]
あざます！！
修正します！！
